.eslintrc

```diff
@@ -40,10 +40,7 @@ overrides:
       - "packages/search-metadata-previews/**/*.js"
     rules:
       react/jsx-no-bind: 1
-      react/no-unused-state: 1
-      react/no-access-state-in-setstate: 1
       react/require-default-props: 1
-      react/default-props-match-prop-types: 1
```

Comment:
I think this whole section can go instead because:

* In your PR, Vraja removed the rules for the search-metadata-previews package in [this commit](https://github.com/Yoast/wordpress-seo/pull/20709/commits/507c113e9a07bdd26f598eafd74194b955bd5926).
* In the feature branch, Vraja removed the rules for the replacement-variable-editor package in [this commit](https://github.com/Yoast/wordpress-seo/commit/7a2c11dfaa75ccf4a10e0f5c15af1d907da1efb0).